services/server/test/util/api_util.js

```diff
@@ -29,14 +29,24 @@ async function createGameAndAssert(t, api, userCred, user, gameOptionalInfo = {}
     commitSHA: 'published-test-app', // check the branch for later commits: published-test-app,
     githubURL: 'https://github.com/turnbasedgames/urturn',
   };
+
+  if (gameOptionalInfo.customURL) {
+    gameRaw.customURL = gameOptionalInfo.customURL;
+  }
+
   const authToken = await userCred.user.getIdToken();
   const { data: { game }, status } = await api.post('/game', gameRaw, { headers: { authorization: authToken } });
+
   t.is(status, StatusCodes.CREATED);
   const { urbux, ...publicUser } = user;
   t.deepEqual(game.creator, publicUser);
   t.is(game.activePlayerCount, 0);
   Object.keys(gameRaw).forEach((key) => {
-    t.is(gameRaw[key], game[key]);
+    if (key === 'customURL') {
+      t.regex(game[key], /[-a-z]/);
```

Comment:
this regex looks different than the regex in the `validator` fn. I think not including numbers is okay but we should make it consistent with the validator fn